templates/_catalog/backendframeworks.json

```diff
@@ -26,5 +26,19 @@
       "version": "1.0.3",
       "preview": false
     }
+  },
+  {
+    "name": "Moleculer",
+    "displayName": "Moleculer",
+    "summary": "JavaScript framework",
+    "author": "Various",
+    "order": "1",
+    "licenses": "[Moleculer](https://github.com/moleculerjs/moleculer/blob/master/LICENSE)",
+    "platforms": ["Web"],
+    "languages": ["Any"],
+    "tags": {
+      "version": "0.13.10",
+      "preview": false
```

Comment:
Sorry, could you also change this one to true? Thank you!

Reply:
Done https://github.com/microsoft/WebTemplateStudio/pull/1173/commits/c09d6307b959b4ae9573d06b7bec4f2df9514a40